source/peer-handshake.js

```diff
@@ -47,6 +47,21 @@ Skylink.prototype._doOffer = function(targetMid, iceRestart) {
     }
   }
 
+
+  /* Safari does not support createOffer with (offerToReceiveAudio: 1)
+   * So the following patch is needed until it is fixed in AdapterJS.
+   * AdapterJS ticket : https://github.com/webrtc/adapter/issues/661
+   */
+  if(window.webrtcDetectedBrowser === 'safari' && webrtcDetectedVersion >= 11 
+      && AdapterJS.webrtcDetectedType !== 'plugin'){
```

Comment:
Can we use `if (AdapterJS.webrtcDetectedType === 'AppleWebKit') {` instead? Also avoid using `window.webrtc[DetectedBrowser|DetectedVersion|MinimumVersion]` because we want to avoid the dependency on the `window` but `AdapterJS`; It's neater that way.